test/TestProjects/ResourceClients-LowLevel/ResourceClients-LowLevel.json

```diff
@@ -0,0 +1,142 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-10-19",
+    "title": "LlcResourceClients"
+  },
+  "host": "localhost:3000",
+  "schemes": [
+    "http"
+  ],
+  "produces": [ "application/json" ],
+  "consumes": [ "application/json" ],
+  "paths": {
+    "/": {
+      "get": {
+        "description": "Method that belongs to the top level service",
+        "operationId": "Service_GetParameters",
```

Comment:
It is confusing from a REST API perspective that the method name is plural but it doesn't return a collection.